src/components/HealthCardDataProvider.js

```diff
@@ -67,8 +92,8 @@ const HealthCardDataProvider = ({ healthCardJws, children }) => {
         const status = await issuerVerify(iss, abortController);
         setIssuerVerified(status);
         if (status === true) {
-          getIssuerDisplayName(cardJws, abortController)
-            .then((result) => setIssuerDisplayName(result));
+          // eslint-disable-next-line max-len
+          getIssuerDisplayName(cardJws, abortController).then((result) => setIssuerDisplayName(result));
```

Comment:
We should probably keep this part on two lines as it was before, since this didn't require us disabling lint.

Reply:
Agree with @mickohanlon23, generally want to avoid disabling lint unless no alternative.